<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearSceneButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>570</y>
      <width>441</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить сцену</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>441</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Модель</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Выбор модели</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QComboBox" name="modelsList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>401</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteModelButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>401</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Удалить выбранную</string>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteAllModelsButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>401</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Удалить все модели</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loadModelButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>401</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Загрузить модель</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>111</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Перенос</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QDoubleSpinBox" name="dxModelInput">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>34</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dyModelInput">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>64</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dzModelInput">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>94</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>34</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>dx:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>64</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>dy:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>94</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>dz:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="moveModelButton">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>140</y>
         <width>107</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Перенести</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>200</y>
        <width>111</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Масштабирование</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QDoubleSpinBox" name="kzModelInput">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>94</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>34</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>kx:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="kxModelInput">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>34</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>94</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>kz:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>64</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ky:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="kyModelInput">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>64</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="scaleModelButton">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>140</y>
         <width>107</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Масштабировать</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>200</y>
        <width>111</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Поворот</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QDoubleSpinBox" name="ozModelInput">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>94</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>34</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ox:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="oxModelInput">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>34</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>94</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>oz:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>64</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>oy:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="oyModelInput">
       <property name="geometry">
        <rect>
         <x>38</x>
         <y>64</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="rotateModelButton">
       <property name="geometry">
        <rect>
         <x>2</x>
         <y>140</y>
         <width>107</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Повернуть</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Камера</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>171</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Поворот</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QDoubleSpinBox" name="ozCameraInput">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>94</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>34</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ox:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="oxCameraInput">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>34</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>94</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>oz:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>64</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>oy:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="oyCameraInput">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>64</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QPushButton" name="rotateCameraButton">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>140</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Повернуть</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>171</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Перенос</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QDoubleSpinBox" name="dxCameraInput">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>34</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dyCameraInput">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>64</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>34</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>dx:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>64</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>dy:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="moveCameraButton">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>140</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Перенести</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>211</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Добавление камеры</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QPushButton" name="addCameraButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>191</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить камеру</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>70</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>y:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>40</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="xCameraInput">
       <property name="geometry">
        <rect>
         <x>28</x>
         <y>40</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="yCameraInput">
       <property name="geometry">
        <rect>
         <x>28</x>
         <y>70</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>108</x>
         <y>100</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>oz:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="oxCameraInput_2">
       <property name="geometry">
        <rect>
         <x>134</x>
         <y>40</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="oyCameraInput_2">
       <property name="geometry">
        <rect>
         <x>134</x>
         <y>70</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>108</x>
         <y>40</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ox:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>108</x>
         <y>70</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>oy:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="ozCameraInput_2">
       <property name="geometry">
        <rect>
         <x>134</x>
         <y>100</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Выбор камеры</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QComboBox" name="camerasList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteCameraButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>171</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Удалить выбранную</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loadCameraButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>171</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Загрузить камеру</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
